fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -3228,7 +3228,15 @@ struct BackupSnapshotManifest : BackupTaskFuncBase {
 		}
 
 		Params.endVersion().set(task, maxVer);
-		wait(bc->writeKeyspaceSnapshotFile(files, beginEndKeys, totalBytes));
+
+		DatabaseConfiguration dbConfig = wait(getDatabaseConfiguration(cx));
```

Comment:
nit: We run a transaction above on line 3161, perhaps we should fetch the database config there. There is a version of `getDatabaseConfiguration` that takes a transaction object

Reply:
Sounds good. Thanks for the suggestion.